<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistryProphylaxisPlanningDialog</class>
 <widget class="QDialog" name="RegistryProphylaxisPlanningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запись в Журнал планирования профилактического наблюдения</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblNote">
     <property name="text">
      <string>Приме&amp;чание</string>
     </property>
     <property name="buddy">
      <cstring>edtNote</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSpeciality"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Планируемые периоды визитов</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="CTableView" name="tblVisitPeriods"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblScene">
     <property name="text">
      <string>&amp;Место</string>
     </property>
     <property name="buddy">
      <cstring>cmbScene</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbScene"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSpeciality">
     <property name="text">
      <string>&amp;Специальность</string>
     </property>
     <property name="buddy">
      <cstring>cmbSpeciality</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblProphylaxisPlanningType">
     <property name="text">
      <string>&amp;Тип планирования профилактики</string>
     </property>
     <property name="buddy">
      <cstring>cmbProphylaxisPlanningType</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="CICDCodeEditEx" name="edtMKB"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblDiagnosis">
     <property name="text">
      <string>&amp;Диагноз</string>
     </property>
     <property name="buddy">
      <cstring>edtMKB</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbProphylaxisPlanningType"/>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" rowspan="2" colspan="2">
    <widget class="QTextEdit" name="edtNote"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEditEx</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>tblVisitPeriods</tabstop>
  <tabstop>cmbProphylaxisPlanningType</tabstop>
  <tabstop>cmbScene</tabstop>
  <tabstop>edtMKB</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegistryProphylaxisPlanningDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegistryProphylaxisPlanningDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
